.env.example

```
@@ -36,3 +36,4 @@ SERVER_PORT=
 WEB_SOCKET_PORT=
 CONNECTION_LIMIT=
 WS_MAX_CONNECTION_TTL=
+CONNECTION_LIMIT_PER_IP=
```

nit: let's differentiate the servers
```suggestion
WS_CONNECTION_LIMIT_PER_IP=
```